File: Data Structures & Algorithms/max-water-container/submission-0.java
class Solution {
    public int maxArea(int[] heights) {
        int highest = 0;
        for(int i = 0; i < heights.length; i++) {
            for(int j = i; j < heights.length; j++) {
                int curr = Math.min(heights[i], heights[j]) * (j-i);
                if(curr > highest) highest = curr;
            }
        }
        return highest;
    }
}

File: Data Structures & Algorithms/min-cost-climbing-stairs/submission-0.py
class Solution:
    def minCostClimbingStairs(self, cost: List[int]) -> int:
        
        dp = [cost[0], cost[1]]
        for i in range(2, len(cost)):
            dp.append(min(cost[i]+dp[i-2], cost[i]+dp[i-1]))
        print(dp)
        return min(dp[-2:])
